<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tcp_Client</class>
 <widget class="QWidget" name="Tcp_Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>621</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lblIP">
      <property name="text">
       <string>IP地址：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txtIP"/>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="lblPort">
      <property name="text">
       <string>端口号：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="txtPort"/>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="btn_Connection">
      <property name="text">
       <string>连接</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="5">
     <widget class="QTextEdit" name="textEdit_Rev"/>
    </item>
    <item row="2" column="0" colspan="5">
     <widget class="QTextEdit" name="textEdit_Send"/>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="btn_Send">
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
